packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -14,3 +14,15 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-prorated-upgrade = Prorated Upgrade
+
+## Product name and frequency of payment for subscription plan change
+
+# Variables:
+#   $name (String) - Name of the upgraded product
```

Comment:
```suggestion
## Checkout line item for subscription plan change listing the product name and frequency of payment 
## For example, a Mozilla VPN subscription charged monthly would appear as: Mozilla VPN (Monthly)
## Variables:
##   $name (String) - Name of the upgraded product (e.g. Mozilla VPN)

```
Individual comments (#) would only appear with the first string immediately after it, so moving the variable explanation to the group comment. 
Also added some additional context to the comment to make it easier for translators to picture the context.

Reply:
Updated as per your suggestions! Thank you the additional info ✍️ . Good to know. Let me know!